.github/workflows/bulloak-check.yml

```diff
@@ -0,0 +1,32 @@
+name: "Check with Bulloak that Solidity tests conform to BTT spec"
+
+on:
+  workflow_call:
+    inputs:
+      skip-modifiers:
+        default: true
+        required: false
+        type: "boolean"
+
+      tree-path:
+        default: "test/**/**.tree"
+        required: false
+        type: "string"
+
+jobs:
+  bulloak-check:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Check out the repo"
+        uses: "actions/checkout@v4"
+
+      - name: "Install and run Bulloak"
+        uses: "smol-ninja/bulloak-toolchain@v1.0.0"
```

Comment:
Can you add a `v1` tag in `bulloak-toolchain` so that we don't have to bump this each time there is a minor version update?

You can see how I to do this here:

https://github.com/PaulRBerg/foundry-multibuild

Reply:
Thanks for sharing the solution. I was looking for it. Fixed in https://github.com/sablier-labs/reusable-workflows/pull/11/commits/9762eebb306996f0464a2087df4757a1e8967023.